app/assets/stylesheets/manage/admins.scss

```diff
@@ -7,6 +7,10 @@
   /*float: right;*/
 }
 
+.future-features {
+  display:none;
+}
+
```

Comment:
@Iulia-Soimaru so this is just a thing i'm adding to hide notes for future things i'd like to add into the views